qri/dataset.py

```diff
@@ -131,6 +135,13 @@ def meta(self):
         self._ensure_populated()
         return self.meta_component
 
+    @property
+    def title(self):
```

Comment:
So I avoiding this:

https://github.com/qri-io/qri-python/pull/37#discussion_r497213156

by doing this awful thing instead. Not sure which you deem worse. Or if you have a better idea.

Reply:
I suppose I should write a test for this if it's a keeper.